In-house patch to remove unnecessary dependencies from Keystone's
requirements files. The specific reasons are as follows:

oauth2		Not applicable to Solaris (being replaced with oauthlib)
		(upstream commit bed88a2e724f5f23a1c839b7872b1bc56f059df5)

pam		No longer applicable
		(upstream commit ff1b41c15b2d65443234ccdb60565d6f6709f879)

--- keystone-2013.2.3/keystone.egg-info/requires.txt.orig	2014-04-03 11:49:55.000000000 -0700
+++ keystone-2013.2.3/keystone.egg-info/requires.txt	2014-05-24 23:09:25.817082422 -0700
@@ -1,5 +1,4 @@
 pbr>=0.5.21,<1.0
-pam>=0.1.4
 WebOb>=1.2.3,<1.3
 eventlet>=0.13.0
 greenlet>=0.3.2
@@ -15,5 +14,4 @@
 python-keystoneclient>=0.3.2
 oslo.config>=1.2.0
 Babel>=1.3
-oauth2
-dogpile.cache>=0.5.0
\ No newline at end of file
+dogpile.cache>=0.5.0

--- keystone-2013.2.3/requirements.txt.orig	2014-04-03 11:47:39.000000000 -0700
+++ keystone-2013.2.3/requirements.txt	2014-05-24 23:09:35.044573447 -0700
@@ -1,6 +1,5 @@
 # keystone dependencies
 pbr>=0.5.21,<1.0
-pam>=0.1.4
 WebOb>=1.2.3,<1.3
 eventlet>=0.13.0
 greenlet>=0.3.2
@@ -16,5 +15,4 @@
 python-keystoneclient>=0.3.2
 oslo.config>=1.2.0
 Babel>=1.3
-oauth2
 dogpile.cache>=0.5.0
